core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -351,6 +351,8 @@ object SparkSubmit {
       OptionAssigner(args.ivyRepoPath, STANDALONE, CLUSTER, sysProp = "spark.jars.ivy"),
       OptionAssigner(args.driverMemory, STANDALONE, CLUSTER, sysProp = "spark.driver.memory"),
       OptionAssigner(args.driverCores, STANDALONE, CLUSTER, sysProp = "spark.driver.cores"),
+      OptionAssigner(args.executorCores, STANDALONE, ALL_DEPLOY_MODES, 
+        sysProp = "spark.executor.cores"),
```

Comment:
this is in the wrong place because it's not just standalone cluster mode. It should be under `// Other options` in L378